docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick start
+
+## What you'll do
+
+- Setup a GitHub repository
+- Activate the GitHub APIs related to self-hosted runner
+- Deploy the [GitHub runner charm](https://charmhub.io/github-runner)
+- Ensure GitHub repository setting are secure
+- Run a simple workflow on the self-hosted runner
+
+## Requirements
+
+- GitHub Account.
+- Juju 3 installed.
+- Juju controller on OpenStack, and a juju model.
+
+For more information about how to install and use Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
+
+## Steps
+
+### Create GitHub repository
+
+The GitHub self-hosted runner spawned by the charm needs to connect to a GitHub repository or organization. GitHub repositories are used as it is simpler to manage.
+
+To create a GitHub repository, log in to [GitHub](https://github.com) with your GitHub Account and follow the instruction [here](https://docs.github.com/en/get-started/quickstart/create-a-repo#create-a-repository). For this tutorial, create a public repository as [GitHub branch protection rules is only available in public repository with GitHub Free tier](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule).
+
+### Activate GitHub APIs related to self-hosted runner
+
+***This must be done for GitHub runner charm to function correctly.***
+
+The GitHub runner charm uses GitHub APIs related to self-hosted runners. Some of the APIs will only be functional after a self-hosted runner registration token is requested for the repository for the first time. If a registration token is requested for the repository before then this can be skipped.
```

Comment:
s/uses/relies on?
s/related to self-hosted runners/for self-hosted runners
What do you mean by "If a registration token is requested for the repository before then this can be skipped."?

Reply:
Removed "If a registration token is requested for the repository before then this can be skipped.". This is repeated information.